python/tvm/meta_schedule/testing/tune_utils.py

```diff
@@ -192,3 +192,50 @@ def f_time_per_layer(
             )
 
     return f_time_per_layer
+
+
+def create_computer(backend: str) -> Callable:
```

Comment:
computer sounds like a weird name, do you have any better proposal?

Reply:
What about `calculator`? The point here is to show this function computes the results for the given workload.